src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -283,6 +284,7 @@ public Object parse(@NotNull final MinecraftServer server, @NotNull final IComma
      * We are returning null to indicate that no boolean value could be parsed.
      */
     @SuppressWarnings({"squid:S2447"})
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S2447' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)